test/ILLink.RoslynAnalyzer.Tests/TestCaseUtils.cs

```diff
@@ -2,22 +2,40 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.IO;
 using System.Linq;
-using System.Text;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
 using Xunit;
 
 namespace ILLink.RoslynAnalyzer.Tests
 {
 	public abstract class TestCaseUtils
 	{
+		public static readonly ReferenceAssemblies Net6PreviewAssemblies =
+			new ReferenceAssemblies (
+				"net6.0",
+				new PackageIdentity ("Microsoft.NETCore.App.Ref", "6.0.0-preview.5.21224.4"),
```

Comment:
I'm not very familiar with ReferenceAssemblies creation but I assume the package identity version doesn't need to be maintained unless we actually need newer ReferenceAssemblies, we just took one version and rely on it even if the linker global.json changes, right?

Reply:
Yup, that's the idea. Unless there's a breaking change in the reference assemblies, we shouldn't care which exact version we use.